src/shipit_signoff/README.md

```diff
@@ -4,7 +4,8 @@
     pip install -r requirements.txt -r requirements-dev.txt -c requirements_frozen.txt 
     pip install -e . -c requirements_frozen.txt 
     Fill in `client_secrets.json.tmpl` as `client_secrets.json` for auth0 connections
-    python3 shipit_signoff/__init__.py
+    Set the environment variables APP_URL DATABASE_URL
```

Comment:
I might be better to point to common place where we document how to get started with development -> https://docs.mozilla-releng.net/contributing/developing.html

And merge your documentation there.

I suppose then *Set the environment variables APP_URL DATABASE_URL* can be replaces with *Set the environment variables that are marked as required in ``settings.py``*